dfktools/duel/duel.py

```diff
@@ -0,0 +1,198 @@
+from web3 import Web3
+
+CONTRACT_ADDRESS = '0xE97196f4011dc9DA0829dd8E151EcFc0f37EE3c7'
```

Comment:
should probably move this to its own constants file

Reply:
I follow the project structure where the contract address is stored in each respectively folder python file